tutor/templates/local/docker-compose.yml

```diff
@@ -57,10 +57,6 @@ services:
       - bootstrap.memory_lock=true
       - discovery.type=single-node
       - "ES_JAVA_OPTS=-Xms{{ ELASTICSEARCH_HEAP_SIZE }} -Xmx{{ ELASTICSEARCH_HEAP_SIZE }}"
-    ulimits:
```

Comment:
Your change will have the effect of disabling ulimits in production, but that's not what we want, right?

Reply:
I'm not sure if that's the result. Please let me know how did you find that?